packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -72,7 +65,7 @@ const AssetSummary = () => {
                     ) : null}
                     <Popover
                         alignment={isMobile() ? 'top' : 'left'}
-                        message={is_eu_popover_text}
+                        message={popover_text}
```

Comment:
```suggestion
                        message={localize('Total assets in all your accounts')}
```
Just inline the message since it is not used anywhere else